bower.json

```diff
@@ -9,8 +9,6 @@
     "lib"
   ],
   "dependencies": {
-    "excanvas": "*",
-    "flot": "0.8.3",
     "SideBySideImproved": "https://github.com/pkoltermann/SideBySideImproved.git"
```

Comment:
Noticing that there is a package.json in the `dev` branch https://github.com/pkoltermann/SideBySideImproved/tree/dev. Possible that it might work, but a fork to the WET org + adding the `package.json` is probably "safer"